editor/editor_inspector.cpp

```diff
@@ -3097,14 +3104,9 @@ void EditorInspector::update_tree() {
 		}
 	}
 
-	// Only hide plugins if we are not editing any object.
-	// This should be handled outside of the update_tree call anyway (see EditorInspector::edit), but might as well keep it safe.
-	_clear(!object);
-
-	if (!object) {
-		get_root_inspector()->set_follow_focus(true);
-		return;
-	}
+	// The call here is for the edited object that has not changed, but the tree needs to be updated (for example, the object's property list has been modified).
+	// Since the edited object has not changed, there is no need to hide the plugin at this time.
+	_clear(false);
```

Comment:
Since the tree is now updated by rebuilding it, `_clear()` is necessary. Plugins are hidden when changing the edited object via `EditorInspector::edit()`.